.github/workflows/sage-test.yml

```diff
@@ -0,0 +1,149 @@
+name: Sage Install Tests
+on:
+  push:
+    paths:
+      - '.github/workflows/sage-test.yml'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'private/scripts/**'
+jobs:
+  test:
+    name: Sage Install Tests
+    env:
+      TERM: xterm-256color
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        php-version: [8.0, 8.1, 8.2, 8.3]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install SSH keys
+        uses: webfactory/ssh-agent@v0.8.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+      - name: Get and Set PR number
+        id: pr
+        run: |
+          pr_num=$(gh pr view --json number -q .number || echo "")
+          echo "pr_num=$pr_num" >> $GITHUB_ENV
```

Comment:
nit: env variables should be all caps

Reply:
yeah, but if I do that, it wouldn't match with the other variables on line 55-57 (https://github.com/pantheon-systems/wordpress-composer-managed/pull/113/files#diff-2faa71adfe1b6b12a9b18a4d722c7463005f533d54912c2154589a1d6b91b3bcR55-R57) or I would need to do `pr_num=$(echo $PR_NUM)`